laravel/app/Models/User.php

```diff
@@ -40,4 +41,12 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function get_post()
```

Comment:
has_manyの場合は複数形の方が良いのと、get_は不要です
```suggestion
    public function posts()
```